.github/workflows/release.yaml

```diff
@@ -10,6 +10,10 @@ on:
 
```

Comment:
What is the best way to test this job? Manually?

Reply:
you can see at the bottom of this file "Test Windows Installer (Windows)" which is similar to what was in the msi.yaml. Assuming msi.yaml -> packaging.yaml works on commit, we should expect low risk doing similarly here for the other two

Reply:
once that's in, the next question is.. would it work on release? One key is that the release.yaml results in a *draft* release not a published one. What I've done in the past is merged things and cut a release tag for the next one. If there was a problem, push a fix to master, delete the release and the tag and push the tag again. Once the release.yaml e2e tests work (via a successful draft release), then it is safe to publish a real release.. either the one just tested or drop the release and push it again.  make sense?